### DISCUSSION TITLE
Styling and Layout Consistency

### DISCUSSION THREAD

**Discussion by ZiyangWang001007081** [ZiyangWang001007081/Tafe230243#4] [Q&A] [answered]
Hi Team,

As part of the Currency Conversion Calculator's XAML page, I'm following the layout and styling of MainMenu XAML page. Should we aim for a consistent look and feel across all calculator features, including the Mortgage Calculator? Are there specific color schemes or design elements we want to maintain throughout the application for a cohesive user experience? 

Please let me know. Thank you.

**Comment by Abhishek-Sathe-Tafe** [comment, marked as answer]
Yeah, I agree. I suggest we go with beige background & default button/label/textbox style & format. Let me know what you think

Cheers!
Abhi
